jest.config.js

```diff
@@ -10,11 +10,13 @@ module.exports = {
     "!**/*.test.{js,jsx}",
     "!**/node_modules/**",
     "!**/vendor/**",
-    "!src/server.js",
+    "!src/bin/server.js",
+    "!src/routes/index.js",
+    "!**/public/**"
   ],
   coverageThreshold: {
     global: {
-      branches: 80,
```

Comment:
We want to keep our quality gates tight, ie keep this at 80, if this gate is failing then we either need to write a quick test or exclude that code from coverage. Its generally easier to keep coverage high from the outset than to raise it later.